src/deploy-actions.js

```diff
@@ -160,6 +165,15 @@ async function deployWsk (scriptConfig, manifestContent, logFunc, filterEntities
     namespace: scriptConfig.ow.namespace
   }
 
+  const lastDeployedActionsPath = path.join(scriptConfig.root, 'dist', 'last-deployed-actions.json')
+  let lastDeployData = {}
+  if (fs.existsSync(lastDeployedActionsPath)) {
+    lastDeployData = await fs.readJson(lastDeployedActionsPath)
+  } else {
```

Comment:
remove unused else

Reply:
done, I chose to output something in this case so users can know it is happening.